### PR TITLE
Avoid pushing duplicate keys

### PR DESCRIPTION
**Pull request by jasl**
This is a small improvement that avoiding pushing duplicate key into `trusted_keys` or `fetched_keys`

### CODE REVIEW

**Comment by jasl** [comment]
This is a small improvement, does it need a changelog and tests?

**Comment by mattbrictson** [comment]
Thanks for the PR! Is this a performance improvement, a bug fix, or just code polish?

File: lib/capistrano/configuration/variables.rb [hunk 1]
```diff
@@ -35,15 +35,15 @@ def untrusted!
...
+        @trusted_keys << key if trusted? && !@trusted_keys.include?(key)
         remember_location(key)
         values[key] = block || value
         trace_set(key)
         values[key]
       end
 
       def fetch(key, default=nil, &block)
-        fetched_keys << key
+        @fetched_keys << key unless @fetched_keys.include?(key)
```
Comment (mattbrictson) [inline lib/capistrano/configuration/variables.rb]: Why did you add the `@`?

Comment (jasl) [inline lib/capistrano/configuration/variables.rb]: I think follow style of `@trusted_keys` should better

**Comment by jasl** [comment]
I think this is a bug fix.

there is a sample
```
[4] pry(main)> require 'capistrano/configuration/variables'
=> true
[5] pry(main)> vars = Capistrano::Configuration::Variables.new
=> #<Capistrano::Configuration::Variables:0x007fb939e3a2c0 @fetched_keys=[], @locations={}, @trusted=true, @trusted_keys=[], @values={}>
[6] pry(main)> vars.set :foo, 'bar'
=> "bar"
[7] pry(main)> vars
=> #<Capistrano::Configuration::Variables:0x007fb939e3a2c0 @fetched_keys=[:print_config_variables], @locations={:foo=>["(pry):6:in `__pry__'"]}, @trusted=true, @trusted_keys=[:foo], @values={:foo=>"bar"}>
[8] pry(main)> vars.set :foo, 'foo'
=> "foo"
[9] pry(main)> vars
=> #<Capistrano::Configuration::Variables:0x007fb939e3a2c0
 @fetched_keys=[:print_config_variables, :print_config_variables],
 @locations={:foo=>["(pry):6:in `__pry__'", "(pry):8:in `__pry__'"]},
 @trusted=true,
 @trusted_keys=[:foo, :foo],
 @values={:foo=>"foo"}>
```

you can see `@trusted_keys=[:foo, :foo]` the key added double

**Comment by mattbrictson** [comment]
OK, thanks for the explanation. I agree this is minor enough to be merged without a test or changelog entry. Could you undo the `@` change per my code comment? Then I'll merge. Thanks!

**Comment by jasl** [comment]
reverted, sorry for delay

**Comment by mattbrictson** [comment]
Great, thanks! 🙇
